src/main/java/com/ibm/devops/connect/CloudPublisher.java

```diff
@@ -438,35 +480,35 @@ public static boolean testConnection(String syncId, String syncToken, String bas
 
             if (response.getStatusLine().toString().contains("200")) {
                 // get 200 response
-                log.info("Connected to Velocity service successfully.");
+                log.info("Connected to Velocity service successfully for: "+baseUrl);
```

Comment:
```suggestion
                log.info("Connected to Velocity service successfully for: " + baseUrl);
```
same spacing should be for all below lines too